File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-1.java
class Solution {
    public int findMin(int[] nums) {
        int ans = nums[0];
        for(int i=0; i<nums.length-1; i++){
            if(nums[i+1]<nums[i]){
                ans = nums[i+1];
                break;
            }
        }
        return ans;
    }
}

File: Data Structures & Algorithms/simplify-path/submission-1.py
class Solution:
    def simplifyPath(self, path: str) -> str:
        stringList = path.split("/")
        print(stringList)
        stack = []
        for string in stringList:
            if string=="" or string==".":
                continue
            elif string=="..":
                if stack:
                    stack.pop()
            else:
                stack.append(string)
            
        ans = "/" + "/".join(stack)
        return ans